packages/export-cli/src/gulp/lint.js

```diff
@@ -4,15 +4,11 @@ const pump = require('pump');
 const { src } = require('gulp');
 const completeTask = require('@parameter1/base-cms-cli-utils/task-callback');
 
-module.exports = (cwd, options) => (cb) => {
+module.exports = (cwd, options, ignore = ['**/node_modules/**/*']) => (cb) => {
   pump([
-    src(['**/*.js'], { cwd }),
+    src(['**/*.js'], { cwd, ignore }),
     cache('basecms-exports-lint-js'),
     eslint(options),
-    eslint.results((results, lintCb) => {
```

Comment:
I honestly don't remember what the results callback (`lintCb`) was being invoked for... that being said, can you better explain why this call was removed?

Reply:
https://github.com/adametry/gulp-eslint/tree/v5.0.0#eslintresultsaction using the second form. This was added to allow calling the user-supplied `cb` function after the lint was complete, but it's unnecessary since `completeTask` below *also* calls the `cb` function once the stream is complete.